### PR TITLE
boto support

### PR DESCRIPTION
**Pull request by joshk0**
Support both boto and rsync storage backends in git-fat so we can store our big binaries on S3/GCS.

[Review on Reviewable](https://reviewable.io/reviews/pippio/git-fat/1)

### CODE REVIEW

**Comment by joshk0** [comment]
@jgraettinger Please have a look too if you're interested.

File: git-fat [hunk 1]
```diff
@@ -2,6 +2,8 @@
 # -*- mode:python -*-
 
 from __future__ import print_function, with_statement
+from boto.s3.connection import S3Connection
+from boto.s3.key import Key
```
Comment (hawka) [inline git-fat]: can we add a requirements.txt if we're bringing in extra dependencies?

Comment (joshk0) [inline git-fat]: Sure, though you'd still have to manually do pip install -r requirements.txt

(Also please use reviewable for CRs)

**Comment by hawka** [comment]
I'm a little sad that the original is so non-PEP8 compliant, but your changes lgtm.

**Pull request closed without merging**